users/users-model.js

```diff
@@ -0,0 +1,28 @@
+const db = require('../data/dbConfig.js');
+
+module.exports = {
+  add,
+  find,
+  findBy,
+  findById,
+};
+
+function find() {
+  return db('users').select('id', 'username');
+}
+
+function findBy(filter) {
+  return db('users').where(filter);
+}
+
+async function add(user) {
+  const [id] = await db('users').insert(user);
```

Comment:
There is nothing wrong with this line for sqlite3 but wanted to warn you that destructing the id from an array, `[id]`, will throw an error when you switch to a PostgreSQL database